File: Solution_Practice_I/python/prob4_11.py
while True:
    num=int(input("Ingrese cuantos numeros ver:"))
    if num>=0:
        break
binacci=1
aux=-1
sw=True
aux2=0
while num>0:
    if sw:
        binacci+=aux
        aux=binacci-aux
        print(binacci, end=" ")
    else:
        while True:
            aux2+=1
            cont=0
            for i in range(1, aux2+1):
                if aux2%i==0:cont+=1
            if cont==2:
                print(aux2, end=" ")
                break
    num-=1
    sw=not sw

File: Solution_Practice_I/java/prob4_11.java
import java.util.Scanner;
public class prob4_11  {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int aux=-1,aux2=0,num,cont,binacci=1;
        boolean sw=true;
        do {
            System.out.println("Ingrese cuantos numeros ver: ");
            num=input.nextInt();
        } while (num<0);
        while (num>0){
            if (sw){
                binacci+=aux;
                aux=binacci-aux;
                System.out.print(binacci+" ");
            }else{
                do {
                    aux2+=1;
                    cont=0;
                    for (int i=1;i<=aux2;i++){
                        if(aux2%i==0)cont+=1;
                    }
                } while (cont!=2);
                System.out.print(aux2+" ");
            }
            sw=!sw;
            num-=1;
        }


        input.close();
    }
}